### PR TITLE
Update post-install.txt

### PR DESCRIPTION
**Pull request by stephanvierkant**
| Q             | A
| ------------- | ---
| License       | MIT

Since `.env` is supposed to be committed to the shared repository (not "gitignored"), you shouldn't change `.env` but only your local environment variables, as per https://symfony.com/doc/current/configuration/dot-env-changes.html

### CODE REVIEW

File: doctrine/doctrine-bundle/1.6/post-install.txt [hunk 1]
```diff
@@ -2,7 +2,7 @@
 <bg=blue;fg=white> Database Configuration </>
 <bg=blue;fg=white>                        </>
 
-  * Modify your DATABASE_URL config in <fg=green>.env</>
+  * Modify your DATABASE_URL config in <fg=green>.env.local</>
```
Comment (weaverryan) [inline doctrine/doctrine-bundle/1.6/post-install.txt]: Yes, but there may *not* be a `.env.local` file. Perhaps we should add a `.env.local` to the `symfony/flex` recipe?

Comment (Pierstoval) [reply doctrine/doctrine-bundle/1.6/post-install.txt]: And add a comment in it to explain how to use it with custom and platform-specific changes 👍

**Comment by dunglas** [comment]
I would keep this message as is (and not introduce a `.env.local` file), because:

* the referenced file already contains a detailed explanation of environments work: https://github.com/symfony/recipes/blob/master/symfony/flex/1.0/.env#L4-L7
* it's up to the user to update the appropriate file (for instance, Docker and VM users will edit `.env` directly, while users installing the project locally on a host machine will prefer using `.env.local`)

**Comment by nicolas-grekas** [comment]
The file that needs to be changed is *really* `.env`, especially after https://github.com/symfony/recipes/pull/674
Thanks for proposing.

**Pull request closed without merging**